modules/ztp-low-latency.adoc

```diff
@@ -1,25 +1,22 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/ztp-deploying-disconnected.adoc
+// * scalability_and_performance/ztp-far-edge/ztp-reference-cluster-configuration-for-vdu.adoc
 
 :_content-type: CONCEPT
-[id="ztp-low-latency-for-distributed-units-dus_{context}"]
-
-= Low latency for distributed units (DUs)
-
-Low latency is an integral part of the development of 5G networks. Telecommunications networks require as little signal delay as possible to ensure quality of service in a variety of critical use cases.
+[id="ztp-low-latency_{context}"]
+= Running low-latency applications on {product-title}
```

Comment:
You could change "low-latency" to "low latency" for consistency with other references in this PR.